app/views/sign_up/completions/show.html.erb

```diff
@@ -1,35 +1,33 @@
 <% title @presenter.heading %>
-<div class="clearfix col-10 sm-col-12 margin-x-auto">
-  <div class="center">
-    <%= image_tag asset_url(@presenter.image_name), width: 140, alt: '', class: 'margin-bottom-2' %>
-  </div>
+<div class="center">
+  <%= image_tag asset_url(@presenter.image_name), width: 140, alt: '', class: 'margin-bottom-2' %>
```

Comment:
Is this an informative image? I see the alt tag is empty.

Reply:
@peggles2 thanks for flagging! Is this covered in LG-5774? I think someone should take a closer look to determine if this and similar images are decorative or not